Index: todo_project/todo_app/views.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.shortcuts import render\r\n\r\n# Create your views here.\r\nfrom django.shortcuts import render, redirect\r\nfrom .models import Task\r\n\r\ndef task_list(request):\r\n    tasks = Task.objects.all()\r\n    return render(request, 'task_list.html', {'tasks': tasks})\r\n\r\ndef task_add(request):\r\n    if request.method == 'POST':\r\n        title = request.POST.get('title')\r\n        description = request.POST.get('description')\r\n        Task.objects.create(title=title, description=description)\r\n        return redirect('task_list')\r\n    return render(request, 'task_add.html')\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/todo_project/todo_app/views.py b/todo_project/todo_app/views.py
--- a/todo_project/todo_app/views.py	
+++ b/todo_project/todo_app/views.py	
@@ -15,3 +15,6 @@
         Task.objects.create(title=title, description=description)
         return redirect('task_list')
     return render(request, 'task_add.html')
+
+def homepage(request):
+    return render(request, 'homepage.html')
\ No newline at end of file
